src/Credentials/CredentialProvider.php

```diff
@@ -358,6 +440,9 @@ private static function remoteCredentialProviders(array $config = [])
         if (!empty(getenv(EcsCredentialProvider::ENV_URI))) {
             $providers['ecs'] = self::ecsCredentials($config);
         }
+        $providers['process_credentials'] = self::process();
+        $providers['process_config']
```

Comment:
Missed the style last go-around, but the SDK tries to follow [PSR-2](https://www.php-fig.org/psr/psr-2/#46-method-and-function-calls) when possible for function calls, which would look like:
```
$providers['process_config'] = self::process(
    'profile_default',
    self::getHomeDir() . '/.aws/config'
);
```

Reply:
Ok.  Changed to this style.